app/api/dao/user.py

```diff
@@ -50,6 +51,11 @@ def create_user(data):
         if 'available_to_mentor' in data:
             user.available_to_mentor = data['available_to_mentor']
 
+        gravatar = Gravatar(email)
+        gravatar_url = gravatar.get_image(size=512)
```

Comment:
This is an unnecessary assignment since we are not using `gravtar_url` again and any future references will be via `user.photo_url`, so we can directly assign `gravatar.get_image(size=512)` to user.photo_url

Reply:
Okay, fixed it:)